docs/index.md

```diff
@@ -73,13 +73,21 @@ connection "kubernetes" {
   # Specify a context other than the current one.
   # config_context = "minikube"
 
+  # Specify the custom resource definitions for which the dynamic tables will be created.
+  # The custom_resource_tables list may include wildcards (e.g. *, ip*, storagestates.migration.???.io), singular names or the full name.
+  # By default plugin will create the dynamic tables for all the available custom resource definitions.
+  # The plugin will not create dynamic tables if custom_resource_tables is empty or not set.
+  # custom_resource_tables = ["certificate","ip*","storagestates.migration.k8s.io"]
+  custom_resource_tables = ["*"]
+
   # If no kubeconfig file can be found, the plugin will attempt to use the service account Kubernetes gives to pods.
   # This authentication method is intended for clients that expect to be running inside a pod running on Kubernetes.
 }
 ```
 
 - `config_context` - (Optional) The kubeconfig context to use. If not set, the current context will be used.
-- `config_path` - (Optional) The kubeconfig file path. If not set, the plugin will check `~/.kube/config`. Can also be set with the `KUBE_CONFIG_PATHS` or `KUBERNETES_MASTER` environment variables. 
+- `config_path` - (Optional) The kubeconfig file path. If not set, the plugin will check `~/.kube/config`. Can also be set with the `KUBE_CONFIG_PATHS` or `KUBERNETES_MASTER` environment variables.
+- `custom_resource_tables` - (Optional) The custom resource definitions to use for the dynamic tables. By default plugin will load the dynamic tables for all the available custom resource definitions. If not set, the plugin will not create any dynamic tables.
```

Comment:
We should be careful with language like `By default the plugin will load the dynamic tables for all ...`, since the behaviour is based off of the `kubernetes.spc` file, which happens to have `custom_resources_tables = ["*"]` uncommented, but if a user is creating their own `kubernetes.spc` file and doesn't set `custom_resource_tables`, the plugin will not create them.

Reply:
I have removed the `By default part..`. will keep it as -
`custom_resource_tables` - (Optional) The custom resource definitions to use for the dynamic tables. If not set, the plugin will not create any dynamic tables.